docs/installation/linux_sources.rst

```diff
@@ -137,3 +137,8 @@ follow these steps within a sourced ROS 2 Humble installation:
     .. code-block:: bash
 
         ShapesDemo
+
+.. note::
+   When eProsima Shapes Demo is compiled with ROS 2 and the
+   `Shapes Demo TypeSupport <https://github.com/eProsima/ShapesDemo-TypeSupport>`_, the "Use ROS2 Topics"
+   checkbox (see :ref:`examples_ros2_compatibility`) is enabled by default.
```

Comment:
I don't think we should mention the checkbox here, just say that it will start automatically in ROS 2 mode and maybe that execution mode can be changed.